features/local-storage.html

```diff
@@ -0,0 +1,74 @@
+
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Local Storage & Cookies storing</title>
+</head>
+<body>
+    <h1>Local Storage & Cookies storing</h1>
```

Comment:
1. can we add a description of what this page does / what it is testing exactly?
2. and a button to go back to the index like other pages have